src/lib/hooks.js

```diff
@@ -142,6 +142,16 @@ const api = {
 		parentCall('hideWidget');
 	},
 
+	showPrint() {
+		store.setState({ hidePrint: false });
```

Comment:
What happens if showPrint() is set to true, but the user does not change hidePrint()?  It looks to me like it will try to do both, which is a little weird?  Perhaps what we really want is just a single method for "showPrint()" and it is either true or false?